packages/grafana-runtime/src/index.ts

```diff
@@ -54,3 +54,10 @@ export { setReturnToPreviousHook, useReturnToPrevious } from './utils/returnToPr
 export { setChromeHeaderHeightHook, useChromeHeaderHeight } from './utils/chromeHeaderHeight';
 export { type EmbeddedDashboardProps, EmbeddedDashboard, setEmbeddedDashboard } from './components/EmbeddedDashboard';
 export { hasPermission, hasPermissionInMetadata, hasAllPermissions, hasAnyPermission } from './utils/rbac';
+export { QueryEditorWithMigration } from './components/QueryEditorWithMigration';
+export {
+  type MigrationHandler,
+  instanceOfMigrationHandler,
+  migrateQuery,
+  migrateRequest,
+} from './utils/migrationHandler';
```

Comment:
Don't understand why this component needs to live in the runtime package? Why not `public/app/features/query`?

Reply:
discussed offline, as it is at the moment, the migration HOC needs to be exported for plugins to use it.